uroute/url.py

```diff
@@ -48,7 +48,7 @@ def _init_rules_data(self):
                 self.rules_data = json.load(rules_file)
             log.debug('URL cleaning rules loaded from %r', self.rules_path)
 
-    def clean_url(self, url):
+    def clean_url(self, url, loop = True):
```

Comment:
I wonder if we can't come up with a more descriptive name than `loop` for this new parameter. How about `clean_redirect`?

Would you mind also adding a line to the docstring explaining the parameter and its purpose, please?